src/js/modules/infragistics.ui.editors.js

```diff
@@ -9668,36 +9693,48 @@
 			size: "normal",
 			/* type="string" Applies custom class on the checkbox, so that custom image can be used.
 				The following jQuery classes can be used in addition http://api.jqueryui.com/theming/icons/
+				```
+				//Initialize
+				$('.selector').%%WidgetName%%({
+					iconClass: "ui-icon-check"
+				});
+
+				//Get
+				var iconClass = $(".selector").%%WidgetName%%("option", "iconClass");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "iconClass", "ui-icon-check");
+				```
 			*/
 			iconClass: "ui-icon-check",
 			/* type="number" Gets/Sets tabIndex attribute for the editor input.
-			```
-			  //Initialize
-			  $('.selector').%%WidgetName%%({
-				  tabIndex: 1
-			  });
+				```
+				//Initialize
+				$('.selector').%%WidgetName%%({
+					tabIndex: 1
+				});
 
-			  //Get
-			  var tabIndex = $(".selector").%%WidgetName%%("option", "tabIndex");
+				//Get
+				var tabIndex = $(".selector").%%WidgetName%%("option", "tabIndex");
 
-			  //Set
-			  $(".selector").%%WidgetName%%("option", "tabIndex", 1);
-			```
+				//Set
+				$(".selector").%%WidgetName%%("option", "tabIndex", 1);
+				```
 			*/
 			tabIndex: 0,
 			/* type="bool" Gets/Sets the readonly attribute. Does not allow editing. Disables changing the checkbox state. On submit the current value is sent into the request.
```

Comment:
Yes, you can change the state programmatically. I will add this to the description.